website/content/v1.1/learn-more/discovery.md

```diff
@@ -1,19 +1,20 @@
 ---
 title: "Discovery"
 weight: 90
-description: "Discover how Sidero Labs impliments Talos node discovery."
+description: "Discover how Sidero Labs implements Talos node discovery."
 ---
 
-We maintain a public discovery service whereby members of your cluster can use a common and unique key to coordinate the most basic connection information (i.e. the set of possible "endpoints", or IP:port pairs).
+We maintain a public discovery service whereby members of your cluster can use a shared and unique key to coordinate the most basic connection information (i.e. the set of possible "endpoints", or IP:port pairs).
```

Comment:
this is little confusing, there is no `shared and unique` :sweat_smile: I think a shared service makes sense

Reply:
I assume it's shared within the cluster, but globally unique, yes? If so, I can clarify

Reply:
I think yes, but @Ulexus can clarify

Reply:
yes, it's same key for all cluster nodes, but globally unique, so that other clusters have different keys (and can't see each other information)

Reply:
Clarified